### PR TITLE
LPS-150499

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related Issue: https://issues.liferay.com/browse/LPS-150499

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

File: modules/apps/object/object-api/src/main/java/com/liferay/object/util/ObjectFieldUtil.java [hunk 1]
```diff
@@ -40,24 +83,18 @@ public static ObjectField createObjectField(
 		objectField.setLabelMap(LocalizedMapUtil.getLocalizedMap(label));
 		objectField.setName(name);
 		objectField.setRequired(required);
+		objectField.setObjectFieldSettings(objectFieldSettings);
```
[the remark this answers is not in the document]
Comment (rodrigopaulino) [reply modules/apps/object/object-api/src/main/java/com/liferay/object/util/ObjectFieldUtil.java:86]: Yes! 👍🏻

**Comment by rodrigopaulino** [comment]
ci:test:sf

File: modules/apps/object/object-api/src/main/java/com/liferay/object/util/ObjectFieldUtil.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (rodrigopaulino) [reply modules/apps/object/object-api/src/main/java/com/liferay/object/util/ObjectFieldUtil.java]: Done, @carolmariaabb. Thanks a lot!

**Comment by liferay-forms** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:reevaluate:1297401_2315

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:forward

**Pull request closed without merging**